sdk/attestation/azure-security-attestation/vcpkg/vcpkg.json

```diff
@@ -14,7 +14,7 @@
     {
       "name": "azure-core-cpp",
       "default-features": false,
-      "version>=": "1.2.0"
+      "version>=": "1.5.0"
```

Comment:
crazy, but you also need the change here:
https://github.com/Azure/azure-sdk-for-cpp/blob/main/sdk/attestation/azure-security-attestation/CMakeLists.txt#L25